.github/workflows/changelog-existence.yml

```diff
@@ -43,6 +44,16 @@ on:
         description: Label on the PR that indicates CHANGELOGs are not required.
         type: string
         required: true
+      changes_dir:
+        description: The directory changelogs are expected to be found. By default, this is '.changes'
+        type: string
+        required: false
+        default: '.changes'
```

Comment:
This dir is forced by changie, I'm not aware of being able to override it.
```suggestion
```

Reply:
agreed, will remove